src/main/java/de/tum/in/www1/artemis/service/notifications/GroupNotificationService.java

```diff
@@ -201,6 +202,7 @@ private void notifyGroupsWithNotificationType(GroupNotificationType[] groups, No
                 case DUPLICATE_TEST_CASE -> createNotification((Exercise) notificationSubject, author, group, NotificationType.DUPLICATE_TEST_CASE,
                         (String) typeSpecificInformation);
                 case ILLEGAL_SUBMISSION -> createNotification((Exercise) notificationSubject, author, group, NotificationType.ILLEGAL_SUBMISSION, (String) typeSpecificInformation);
+                default -> throw new UnsupportedOperationException("Unsupported NotificationType: " + notificationType);
```

Comment:
Nice 👍

Reply:
I actually had to add this one because of the new notification type wich is the first one that has no use in the GroupNotificationService the switch case complained about a missing type and this default catch fixed the complaint^^